src/create_torrent.cpp

```diff
@@ -372,7 +372,7 @@ namespace {
 			info_hash
 		};
 
-		storage_holder storage = disk_thread->new_torrent(std::move(params)
```

Comment:
this makes sense. and in fact, `new_torrent()` should probably take this by `const&` as well